inc/Ui/Search.php

```diff
@@ -584,29 +584,25 @@ protected function getFulltextResultsHTML($data, $highlight)
             if ($cnt !== 0) {
                 $resultHeader[] = $cnt . ' ' . $lang['hits'];
                 if ($num < FT_SNIPPET_NUMBER) { // create snippets for the first number of matches only
-                    $snippet = '<dd>' . ft_snippet($id, $highlight) . '</dd>';
-                    $lastMod = '<span class="search_results__lastmod">' . $lang['lastmod'] . ' ';
+                    $snippet = '<dd class="snippet">' . ft_snippet($id, $highlight) . '</dd>';
+                    $lastMod = $lang['lastmod'] . ' ';
                     $lastMod .= '<time datetime="' . date_iso8601($mtime) . '" title="'.dformat($mtime).'">' . dformat($mtime, '%f') . '</time>';
-                    $lastMod .= '</span>';
                 }
                 $num++;
             }
 
-            $metaLine = '<div class="search_results__metaLine">';
+            $metaLine = '<dd class="lastmod">';
             $metaLine .= $lastMod;
-            $metaLine .= '</div>';
-
+            $metaLine .= '</dd>';
 
             $eventData = [
                 'resultHeader' => $resultHeader,
                 'resultBody' => [$metaLine, $snippet],
                 'page' => $id,
             ];
             trigger_event('SEARCH_RESULT_FULLPAGE', $eventData);
-            $html .= '<div class="search_fullpage_result">';
```

Comment:
What is the motivation behind removing this `div`? I find it helpful for styling purposes. And it _is_ valid HTML per WHATWG Living Standard: https://html.spec.whatwg.org/multipage/grouping-content.html#the-dl-element

Reply:
I know this will be valid in the coming HTML 5.2 spec (and the validator already says it's fine as well). The main reason why I removed it was because I couldn't see it being used anywhere and generally like to keep code clear of redundant things
On the one hand I wouldn't mind moving it back as it can make styling a dl easier, but on the other hand this is a relative simple list and I'm not sure if anyone is ever going to use it.

Reply:
While it is currently not used in the default template, one use-case would be to give the search results a slightly different background color. Or some box-shadow in order to create a more card-like / material appearance. This might be possible without such a wrapping element, but it might be somewhat arcane/non-trivial usage

Reply:
When we stay with a DL I would prefer to keep this div.